src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -186,10 +209,13 @@ public static class EntityTypes
         private static void init() {}
 
         public static final TagKey<EntityType<?>> BOSSES = tag("bosses");
+        public static final TagKey<EntityType<?>> MINECARTS = tag("minecarts");
+        public static final TagKey<EntityType<?>> BOATS = tag("boats");
+        public static final TagKey<EntityType<?>> CAPTURING_NOT_SUPPORTED = tag("capturing_not_supported");
```

Comment:
This should have a javadoc similar to the relocation not supported tag explaining what you mean by "capturing"